spec/attime_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'attime'
+
+describe '#attime' do
+  subject do |example|
+    delta = example.metadata[:description_args][0]
+    CheckGraphite.attime(delta)
+  end
+
+  def timedelta(n)
+    Time.now + n
```

Comment:
Let's hope there's no GC pauses during the test run... since `.attime` takes the current time as second argument I think you should have a `let :now` and use that here and in the `.attime` call to make sure that time doesn't elapse during the test run.